lib/ret_web/controllers/page_controller.ex

```diff
@@ -452,8 +452,23 @@ defmodule RetWeb.PageController do
     Ret.AppConfig.get_config(!!module_config(:skip_cache)) |> generate_config("APP_CONFIG")
   end
 
+  defp escape_themes(%{"themes" => themes_string} = config) do
+    case Poison.decode(themes_string) do
+      {:ok, themes_map} ->
+        Map.put(config, "themes", themes_map)
+
+      _ ->
+        config = Map.put(config, "themes", [])
+        Map.put(config, "themes_failed_to_load", true)
```

Comment:
Do we have something on the front end that reads this value and throws an error?

Reply:
No, but I'll add that.

Reply:
Added https://github.com/mozilla/hubs/pull/4109/commits/515208958049fd77c616ae81e1e273473f6a4984